src/server/services/afval/afvalwijzer.ts

```diff
@@ -145,8 +145,12 @@ function transformFractionData(
       addressCode
     : fractionData.afvalwijzerUrl;
 
+  const afvalwijzerOphaaldagen2 =
+    fractionData.afvalwijzerOphaaldagen2 instanceof Array
```

Comment:
Iets betrouwbaarder is:
```suggestion
    Array.isArray(fractionData.afvalwijzerOphaaldagen2)
```